src/test/java/org/rulez/demokracia/pdengine/testhelpers/InitialContextFactoryMock.java

```diff
@@ -0,0 +1,28 @@
+package org.rulez.demokracia.pdengine.testhelpers;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Hashtable;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.naming.spi.InitialContextFactory;
+
+public  class InitialContextFactoryMock implements InitialContextFactory {
+
+  private final static String KEYSTORE="/pdengine/src/main/resources/keystore.pk12";
```

Comment:
do not use static path. it works without "/pdengine/".

Reply:
OK, fine

Reply:
Actually not. Changed to relative path and then the local docker image build is successful but on shippable it fails.

Reply:
I also have no idea why it fails on shippable. It cannot go to develop this way nevertheless.
Find the right way to reference it. Use the classloader for example.
And put it to the test resources, not the production ones, sorry about that.

Reply:
OK, put to target/test-classes/resources, it works on shippable too (/pdengine bind mount is not available in shippable container).